juri/styles.black.css

```diff
@@ -0,0 +1,68 @@
+html {
```

Comment:
black CSS just need to have variables overrided. You don't need to have a lot of rules here.